habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -295,23 +284,21 @@ def get_candidate_starts(
                 "object_category": category,
                 "view_points": [],
             }
-            if not obj_rearrange_easy:
-                obj_goals.append(obj_goal)
-            else:
-                # use parent object name if it exists
-                rec_name = rec_to_parent_obj[
-                    name_to_receptacle[obj_idx_to_name[i]]
-                ]
-                # otherwise use receptacle name
-                if rec_name is None:
-                    rec_name = name_to_receptacle[obj_idx_to_name[i]]
-                recep_cat = get_rec_category(
-                    rec_name.split(":")[0],
-                    rec_category_mapping=rec_category_mapping,
-                )
-                if recep_cat == start_rec_cat:
-                    obj_goals.append(obj_goal)
-    return obj_goals
+            obj_goals_hard.append(obj_goal)
```

Comment:
Sorry I didnt totally understood what easy and hard signify here?

Reply:
I'm also confused by this.

Reply:
Look [here](https://github.com/facebookresearch/habitat-lab/pull/1280#discussion_r1172689880).